diffsims/generators/rotation_list_generators.py

```diff
@@ -0,0 +1,122 @@
+    return rotation_list
+
+def get_fundemental_zone_grid(space_group_number, resolution):
+    """
+    Creates a rotation list for the rotations within the fundemental zone of a given space group.
+
+    Parameters
+    ----------
+    space_group_number : int
+        Between 1 and 230
+
+    resolution : float
+        The 'resolution' of the grid (degrees)
+
+    Returns
+    -------
+    rotation_list : list of tuples
+    """
+    zone_string = get_proper_point_group_string(space_group_number)
+    raw_grid = create_linearly_spaced_array_in_rzxz(resolution)  # could cut the count down here
+    raw_grid_axangle = raw_grid.to_AxAngle()
+    fz_grid_axangle = reduce_to_fundemental_zone(raw_grid_axangle, zone_string)
+    return _returnable_eulers_from_axangle(fz_grid_axangle,'rzxz',round_to=2)
+
+
+def get_local_grid(center, max_rotation, resolution):
+    """
+    Creates a rotation list for the rotations within max_rotation of center at a given rotation.
+
+    Parameters
+    ----------
+    center : 3 angle tuple
+        The orientation that acts as the center of the grid, specified in the
+        'rzxz' convention (degrees)
+
+    max_rotation : float
+        The largest rotation away from 'center' that should be included in the grid (degrees)
+
+    resolution : float
+        The 'resolution' of the grid (degrees)
+
+    Returns
+    -------
+    rotation_list : list of tuples
+    """
+    raw_grid = _create_advanced_linearly_spaced_array_in_rzxz(resolution,360,max_rotation + 10,360)
+    raw_grid_axangle = raw_grid.to_AxAngle()
+    raw_grid_axangle.remove_large_rotations(np.deg2rad(max_rotation))
+    if not np.all(np.asarray(center) == 0):
+        raw_grid_axangle = rotate_axangle(raw_grid_axangle, center)
+
+    return _returnable_eulers_from_axangle(raw_grid_axangle,'rzxz',round_to=2)
+
+
+def get_grid_around_beam_direction(beam_direction,resolution, angular_range=(0, 360)):
+    """
+    Creates a rotation list of rotations for which the rotation is about given beam direction
+
+    Parameters
+    ----------
+    beam_direction : [x,y,z]
+        A desired beam direction
+
+    resolution : float
+        The 'resolution' of the grid (degrees)
+
+    angular_range : tuple
+        The minimum (included) and maximum (excluded) rotation around the beam direction to be included
+
+
+    Returns
+    -------
+    rotation_list : list of tuples
+    """
+    from itertools import product
+
+    rotation_alpha, rotation_beta = _get_rotation_to_beam_direction(beam_direction)
+    # see _create_advanced_linearly_spaced_array_in_rzxz for details
+    steps_gamma = int(np.ceil((angular_range[1] - angular_range[0])/resolution))
+    alpha = np.asarray([rotation_alpha])
+    beta =  np.asarray([rotation_beta])
+    gamma = np.linspace(angular_range[0],angular_range[1], num=steps_gamma, endpoint=False)
+    z = np.asarray(list(product(alpha, beta, gamma)))
+    raw_grid = Euler(z, axis_convention='szxz')
```

Comment:
Please could you add a comment in here to flag that this function is using a different Euler convention to be efficient and then converting back to the "standard" Euler convention.